build_opts/VEGA_X86

```diff
@@ -1,4 +1,5 @@
 RUBY=y
+NUMBER_BITS_PER_SET=128
```

Comment:
Unrelated to this PR, but there really ought to be a way to configure the bitset at runtime so this isn't necessary.

I don't really like needing to delete my build directory and build from scratch so that scons propogates this variable.

Reply:
That is a good suggestion! I will look into this

Reply:
This would be a great improvement. I do worry that it's something like `typedef bitset uint8_t[NUMBER_OF_BITS_PER_SET/8];` or something like that which will be difficult to change dynamically.

Reply:
There are dynamic_bitset kind of things in external libs (e.g., boost). But that seems like a lot of overhead for one thing.  I wonder if we can get away with vector<bool>..

Reply:
Just FYI, this is defined in `src/mem/ruby/common/Set.hh`. It uses a bitset which requires the size to be known at compile time. https://en.cppreference.com/w/cpp/utility/bitset

We could *possibly* use `vector<bool>` (https://en.cppreference.com/w/cpp/container/vector_bool), but I don't know how we would know dynamically what the max entries per set will be (probably the number of machines used for `NetDest`, but I'm not certain).